src/test/resources/features/UKTRScenarios.feature

```diff
@@ -3,21 +3,35 @@ Feature: UKTR Scenarios
 
   Scenario Outline: Verify that the Group has not enrolled
     Given I have generated a bearer token for an <UserType> and <Enrolment>
-    And I make API call to PLR UKTR
+    And I make API call to <RequestUrl> and <Endpoint> and <PLRID>
     Then I verify the response code is <StatusCode>
     Examples:
-      | UserType     | Enrolment         | StatusCode |
-      | Organisation | with enrolment    | 201        |
-      | Organisation | without enrolment | 401        |
+      | UserType     | Enrolment         | StatusCode | RequestUrl     | Endpoint      |
```

Comment:
where is the PLRID defined here?

Reply:
This is not related to pil-1362, In order to work with the new URL I have just parameterized to pass the scenario.

Reply:
Created a new ticket for refactoring the code -PIL-1516